lib/web/static/js/app.js

```diff
@@ -0,0 +1,81 @@
+const error_alert_markup = `
+<div class="alert alert-danger alert-dismissible fade show" role="alert">
+  Error on the request to get more resources of this state.
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>`
+
+class API {
+  constructor() { this.url = "/" }
+
+  getResourcesForState(list, callback) {
+    let xhr = new XMLHttpRequest();
+    let state = list.dataset.state
+    page++;
+
+    xhr.open('GET', 'machinery/api/'+state+'/resources/'+page);
+    xhr.send(null);
+    xhr.onreadystatechange = function () {
+      const OK = 200;
+      const DONE = 4;
+
+      if (xhr.readyState === DONE) {
+        if (xhr.status === OK) {
+          callback(JSON.parse(xhr.response))
+        } else {
+          list.parentNode.querySelector('.info').innerHTML = error_alert_markup
+        }
+      }
+    }
+  }
+}
+
+const listHolders = document.querySelectorAll('.holder');
+const resourcesLists = document.querySelector('.infinite-list');
+const api = new API();
+let page = 1;
+
+var loadMore = function(state_holder) {
+  api.getResourcesForState(state_holder, function (elements){
+    elements.forEach(function (resource) {
+      friendly_module_name = "lol"
+      template = `
+      <li class="list-group-item resource-item" data-toggle="modal" id="modal_link_${resource.id}" data-target="#modal${resource.id}">
+        ${friendly_module_name} - #${resource.id}
```

Comment:
Possible undefined variable: 'friendly_module_name' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)